playbooks/kind.yaml

```diff
@@ -6,8 +6,6 @@
     - metallb
   vars:
     kind_cluster_name: ansible
-    kind_kubectl_default_namespace: seldon
-
-    # For compatible version of image for each kind version please see https://github.com/kubernetes-sigs/kind/releases
```

Comment:
I'd leave this comment line as it is quite useful to have it for quick reference.